.github/workflows/CI.yml

```diff
@@ -823,31 +838,35 @@ jobs:
             "$HOME/Library/Application Support/com.apple.TCC/TCC.db"
           )
 
-          sqlQuery="INSERT OR IGNORE INTO access VALUES($values);"
-
-          for dbPath in "${dbPaths[@]}"; do
-            echo "Column names for $dbPath"
-            echo "-------------------"
-            sudo sqlite3 "$dbPath" "PRAGMA table_info(access);"
-            echo "Current permissions for $dbPath"
-            echo "-------------------"
-            sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
-            sudo sqlite3 "$dbPath" "$sqlQuery"
-            echo "Updated permissions for $dbPath"
-            echo "-------------------"
-            sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
+          for value in "${values[@]}"; do
+            for dbPath in "${dbPaths[@]}"; do
+              echo "Column names for $dbPath"
+              echo "-------------------"
+              sudo sqlite3 "$dbPath" "PRAGMA table_info(access);"
+              echo "Current permissions for $dbPath"
+              echo "-------------------"
+              sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
+              execute_sql_query "$value" "$dbPath"
+              echo "Updated permissions for $dbPath"
+              echo "-------------------"
+              sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
+            done
           done
 
-      - name: Run tests
+      - name: Run tests directly
         id: test
         timeout-minutes: 10
+        working-directory:
+          /opt/local/var/macports/build/_Users_runner_work_Sunshine_Sunshine_ports_multimedia_Sunshine/Sunshine/work/build/tests
         run: |
-          sudo port test "Sunshine"
+          sudo port install doxygen
+          sudo port install graphviz
+          sudo ./test_sunshine --gtest_color=yes
 
       - name: Test Logs
         if: always()
         run: |
-          logfile="/opt/local/var/macports/logs/_Users_runner_work_Sunshine_Sunshine_ports_multimedia_Sunshine/Sunshine/main.log"
+          logfile="/opt/local/var/macports/build/_Users_runner_work_Sunshine_Sunshine_ports_multimedia_Sunshine/Sunshine/work/build/tests/test.log"
           cat "$logfile"
```

Comment:
This step probably isn't even necessary if we stick with running the test binary directly.

Reply:
I think it's still useful, because I was looking at this file, and it seems to create the BOOST logs from the test run, only the boost stdout without all the other test output, even without the results, but it may be good for debugging.